releasenotes/notes/ddsketch-api-a84ffc0875bbacd6.yaml

```diff
@@ -0,0 +1,5 @@
+---
+upgrade:
+  - "``DDSketch`` attributes ``mapping``, ``store``, ``negative_store``, ``zero_count``, ``relative_accuracy``, ``min`` and ``max`` have been made internal."
```

Comment:
@brettlangdon I'm conflicted as to whether we should say removed vs "made internal". Any thoughts?

Reply:
I think "removed" and "made readonly" reads good?

Reply:
yeah I think I prefer "removed"